Issue #181 add instructions for building the user documentation as html (#180)
code-ready/crc: README.adoc

[[documentation]]
== Documentation

> === Using CodeReady Containers

Reviewer: :+1:

> You can find the raw documentation files link:./docs/source[here]

Reviewer: ```suggestion
You can find the source files for the documentation in the link:./docs/source[docs/source] directory.
```

Author: done

Reviewer: I don't see these changes. Other than that, everything looks good. :+1:

Author: ah, missed that one. done!

> However, to build a nicely formatted html document do this:

Reviewer: ```suggestion
To build the formatted documentation, use the following:
```

Author: done

```bash

> $ cd $HOME

Reviewer: I don't believe this is a necessary step.

Author: ack, it's not, I added that only so the path I mentioned later could be absolote :)

$ git clone https://github.com/code-ready/crc
$ cd crc
$ make build_docs
```

> This will create $HOME/crc/docs/build/master.html which you can view in your browser.

Reviewer: ```suggestion
This will create a [filename]`docs/build/master.html` file which you can view in your browser.
```

Author: done

=== Developing CodeReady Containers

> Developers wanting to work on CodeReady Containers should visit link:./developing.adoc[Developing CodeReady Containers]

Reviewer: ```suggestion
Developers who want to work on CodeReady Containers should visit the link:./developing.adoc[Developing CodeReady Containers] document.
```

Author: done
